packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -84,7 +102,7 @@ export const MafsGraph = (props: MafsGraphProps) => {
                         padding: "25px 25px 0 0",
                         boxSizing: "content-box",
                         marginLeft: "20px",
-                        marginBottom: "20px",
+                        marginBottom: "30px",
                         pointerEvents: props.static ? "none" : "auto",
```

Comment:
We needed a little more breathing room for the labels now that they're within the SVG